SU2_CFD/include/solvers/CFVMFlowSolverBase.inl

```diff
@@ -1260,7 +1259,7 @@ void CFVMFlowSolverBase<V, R>::BC_Sym_Plane(CGeometry* geometry, CSolver** solve
                 GradNormVel[iDim] * UnitNormal[iVar] + GradTangVel[iDim] * Tangential[iVar];
 
         /*--- Set the primitive gradients of the boundary and reflected state. ---*/
-        visc_numerics->SetPrimVarGradient(nodes->GetGradient_Primitive(iPoint), Grad_Reflected);
+        visc_numerics->SetPrimVarGradient(nodes->GetGradient_Primitive(iPoint), CMatrixView<su2double>(Grad_Reflected));
```

Comment:
Matrix types have an explicit conversion to this "matrix view", i.e. you can pass `su2activematrix` to CNumerics instead of a `su2double**`